src/transactions/TxEnvelopeTests.cpp

```diff
@@ -185,6 +185,169 @@ TEST_CASE("txenvelope", "[tx][envelope]")
         }
     }
 
+    SECTION("alternative signatures")
```

Comment:
This is in next commit - 58c084650cc1e4c92c709e2fee0d6fcfc985a9af, I can merge if you want